cupy/linalg/einsum.py

```diff
@@ -187,7 +280,12 @@ def einsum(*operands):
         else:
             converted_inputs.append(cupy.asarray(a, dtype=dtype))
 
-    match = re.match('^([a-zA-Z,]+)(->[a-zA-Z]*)?$', subscripts)
+    subscripts = subscripts.replace('...', '@')
```

Comment:
`...` is replaced with `@`  for simplicity of implementation, right? If so please write a comment about it.

Reply:
Yes, I should add comment.